stable/dex-controller/templates/deployments.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: {{ template "dex-controller.fullname" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "dex-controller.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: {{ .Values.controller.replicas }}
+  selector:
+    matchLabels:
+      control-plane: {{ template "dex-controller.fullname" . }}
+  template:
+    metadata:
+      labels:
+        control-plane: {{ template "dex-controller.fullname" . }}
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        release: {{ .Release.name }}
+        heritage: {{ .Release.Service }}
+    spec:
+      serviceAccountName: {{ template "dex-controller.fullname" . }}-sa
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
```

Comment:
Just making sure this is the level of verbosity that you want

Reply:
Will make it configurable in a subsequent release.